pkg/image-manager/image_builder.go

```diff
@@ -247,12 +248,57 @@ func cleanup(tmpDir string) {
 	os.RemoveAll(tmpDir)
 }
 
-func writeDockerFile(dir, baseImageURL string) error {
-	dockerFileContent := []byte(fmt.Sprintf("FROM %s\n", baseImageURL))
-	if err := ioutil.WriteFile(filepath.Join(dir, "Dockerfile"), dockerFileContent, 0644); err != nil {
-		return errors.Wrap(err, "failed to write Dockerfile")
+func writeSystemDependencies(image *Image) string {
+	dockerFileContent := []string{}
+	for _, pkg := range image.SystemDependencies.Packages {
+		if pkg.Version == "" {
+			dockerFileContent = append(dockerFileContent, fmt.Sprintf("RUN tdnf install -y %s", pkg.Name))
```

Comment:
Is each package a separate layer on purpose here? If not, maybe we could combine it into:
`RUN tdnf install -y package1 package2 && tdnf clean all`

Reply:
I was a little lazy here... I'll fix it.

Reply:
Actually, this is dead code anyway, but the same issue exists in the real code.